src/pip/_internal/commands/list.py

```diff
@@ -310,10 +311,12 @@ def format_for_columns(
         header = ["Package", "Version"]
 
     data = []
-    if options.verbose >= 1 or any(x.editable for x in pkgs):
+    if options.verbose >= 1:
         header.append("Location")
     if options.verbose >= 1:
         header.append("Installer")
+    if any(x.editable for x in pkgs):
+        header.append("Editable project location")
```

Comment:
Yeah, compatibility was the reason I put there. But since we are changing the column output anyway and people should really not be parsing it, lets put it after Version so the verbose mode adds columns and Editable Project Location and Location are next to each other.